common.yml

```diff
@@ -1416,13 +1430,12 @@ buildvariants:
 #     Amazon x86_64 Buildvariants     #
 #######################################
 
-- name: amazonlinux64
+- name: amazon
```

Comment:
did some work to match the server's platform naming here. also removed the largely-redundant `_platform` expansion